validator/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .validator import Validator, validate
+from . import rules_src
```

Comment:
Do we really need this import? If yes why?

Reply:
My bad :)